lib/matplotlib/backends/backend_cairo.py

```diff
@@ -129,6 +129,7 @@ class RendererCairo(RendererBase):
     def __init__(self, dpi):
         self.dpi = dpi
         self.gc = GraphicsContextCairo(renderer=self)
+        self.ctx = self.width = self.height = None
```

Comment:
I'd prefer to init them on separate lines for better readability.

Reply:
sure